test/test_images/performance/main.go

```diff
@@ -112,12 +115,17 @@ func main() {
 		exec = newSenderReceiverExecutor(pacerSpecs, aggCli)
 
 	case "aggregator":
+		var tags []string
```

Comment:
Already declared as a global var.

Reply:
@antoineco This is a new var (string array)

Reply:
Ah you don't parse the flag directly anymore, my bad.